schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json

```diff
@@ -462,6 +476,17 @@
         "InstallerType": {
           "$ref": "#/definitions/InstallerType"
         },
+        "NestedInstallers": {
+          "type": "array",

```

Comment:
We can do all of the multi-installer scenarios by having multiple installers defined at the top level and having all of them reference the same archive, but different files within.

Reply:
> We can do all of the multi-installer scenarios by having multiple installers defined at the top level and having all of them reference the same archive, but different files within.

We can, yes, but then there is a lot of duplication of data. The Installer URL and Sha256 at minimum would be repeated for however many installer nodes there are, which could be many when considering the multiplicative effect of locale and architecture.

In my mind it would make updating manifests more difficult and would complicate the flow of wingetcreate and YamlCreate even further. Whereas, having the multi-installer scenarios handled by an array here would likely result in a cleaner user experience